source/unified-test-format/unified-test-format.rst

```diff
@@ -225,6 +238,9 @@ Test runners MUST support the following types of entities:
 - ClientSession. See `entity_session`_ and `Session Operations`_.
 - GridFS Bucket. See `entity_bucket`_ and `Bucket Operations`_.
 - ChangeStream. See `ChangeStream Operations`_.
+- Event list. The event list MUST store BSON documents. The type of the list
+  itself is not prescribed by this specification. Test runner MAY use a
```

Comment:
> The type of the list itself is not prescribed by this specification

Is this necessary to state? The spec is language-agnostic so I am not sure how much more prescriptive you _can_ be in terms of type beyond saying that this is a list.

Reply:
The following sentence explicitly defines two possible storage implementations - a simple array and an array that synchronizes access.

Reply:
AFAIK, this is the only type in the entity map that doesn't have a prescribed type. Everything else is either a specific driver object or BSON type. The entity map is one of the few places in the spec where we _do_ talk about types, because some languages were unable to implement it as a generic dictionary.

I don't think there's any harm in keeping this note, although see your point that it may be redundant in light of the sentence that follows.